src/vl-map-action.js

```diff
@@ -43,67 +61,76 @@ export class VlMapAction extends vlElement(HTMLElement) {
   /**
    * Zet de kaartlaag.
    *
-   * @param {ol.layer.Layer} layer
+   * @param {Object} layer
```

Comment:
Misschien toch een duiding geven dan dat het hier gaat om de openlayers layer? Ook hieronder

Reply:
Probleem is dat door de manier van importeren in OL6 je niet meer kan verwijzen naar de classes in documentatie, maar ik zal het aanvullen in de documentatie zelf.